app/src/main/java/com/sunasterisk/travelapp/data/LoadDataAsync.kt

```diff
@@ -2,27 +2,23 @@ package com.sunasterisk.travelapp.data
 
 import android.os.AsyncTask
 
-class LoadDataAsync<T, P>(
+class LoadDataAsync<T, P : Any>(
```

Comment:
Mặc định tất cả các lớp đều kế thừa Any
```suggestion
class LoadDataAsync<T, P>(
```

Reply:
Khi em để P không kế thừa Any thì bị lỗi ạ
![Screenshot from 2020-09-13 12-06-23](https://user-images.githubusercontent.com/57834181/93010725-b608b400-f5b9-11ea-9e03-a36f4b3b5e91.png)